public/app/core/components/AppChrome/ReturnToPrevious/ReturnToPrevious.tsx

```diff
@@ -10,15 +10,15 @@ import { t } from 'app/core/internationalization';
 import { DismissableButton } from './DismissableButton';
 
 export interface ReturnToPreviousProps {
-  href: string;
   title: string;
+  href?: string;
```

Comment:
think this should still be required. the component must always be passed an `href`